BootloaderCommonPkg/Library/ShellLib/CmdBoot.c

```diff
@@ -17,6 +17,7 @@
 #include <Guid/OsBootOptionGuid.h>
 #include "Shell.h"
 
+extern UINT8    mCurrentBoot;
```

Comment:
OS Loader uses this value to track which boot option to try next. We can modify it here in the 'boot' command if we want to change the next option.

https://github.com/slimbootloader/slimbootloader/blob/master/PayloadPkg/OsLoader/OsLoader.c#L911